rclpy/src/rclpy/_rclpy.c

```diff
@@ -5093,10 +5094,19 @@ PyMODINIT_FUNC PyInit__rclpy(void)
     return NULL;
   }
 
+  RCLError = PyErr_NewExceptionWithDoc(
+    "_rclpy.RCLError",
+    "Thrown when there is error in calling the rcl layers.",
+    PyExc_RuntimeError, NULL);
+  if(PyModule_AddObject(m, "RCLError", RCLError)){
```

Comment:
There is a linter test failure. It's good to run tests locally before opening PRs to catch these kinds of errors.
```suggestion
  if (PyModule_AddObject(m, "RCLError", RCLError)) {
```